pkgs/top-level/all-packages.nix

```diff
@@ -21970,6 +21970,17 @@ with pkgs;
     go = buildPackages.go_1_18;
   };
 
+  # requires a newer Apple SDK
+  go_1_19 = darwin.apple_sdk_11_0.callPackage ../development/compilers/go/1.19.nix {
+    inherit (darwin.apple_sdk_11_0.frameworks) Foundation Security;
+  };
+  buildGo119Module = darwin.apple_sdk_11_0.callPackage ../development/go-modules/generic {
+    go = buildPackages.go_1_19;
+  };
+  buildGo119Package = darwin.apple_sdk_11_0.callPackage ../development/go-packages/generic {
+    go = buildPackages.go_1_19;
+  };
```

Comment:
Do we really need thi?

Reply:
> Do we really need thi?

If you are asking if we need `buildGo119Package`, as long as we are still using `buildGoPackage` we need a versioned equivalent.